daemon/cluster/executor/container/container.go

```diff
@@ -718,6 +718,34 @@ func (c *containerConfig) applyPrivileges(hc *enginecontainer.HostConfig) {
 			hc.SecurityOpt = append(hc.SecurityOpt, "label=type:"+selinux.Type)
 		}
 	}
+
+	// variable to make the lines shorter and easier to read
+	seccomp := privileges.Seccomp
+	if seccomp != nil {
```

Comment:
Minor nit; might as well keep it fully local to this `if`;

```suggestion
	if seccomp := privileges.Seccomp; seccomp != nil {
```